src/commands/deployWorkspaceProject/deploymentConfiguration/workspace/FilePathsVerifyStep.ts

```diff
@@ -3,50 +3,71 @@
+export abstract class FilePathsVerifyStep extends ExecuteActivityOutputStepBase<WorkspaceDeploymentConfigurationContext> {
+    abstract key: string;
+    abstract fileType: string;
+
+    private configPath: string | undefined;
+
+    public constructor() {
+        super();
+    }
 
-    // Todo: Add logic to verify that the file paths actually exist, if they don't, leave the corresponding context value as undefined
     protected async executeCore(context: WorkspaceDeploymentConfigurationContext, progress: Progress<{ message?: string | undefined; increment?: number | undefined }>): Promise<void> {
         this.options.shouldSwallowError = true;
-        progress.report({ message: localize('verifyingFilePaths', 'Verifying file paths...') });
+        progress.report({ message: localize('verifyingFilePaths', `Verifying file paths...`) });
 
         const rootPath: string = nonNullProp(context, 'rootFolder').uri.fsPath;
 
-        const configDockerfilePath = context.deploymentConfigurationSettings?.dockerfilePath;
-        if (!context.dockerfilePath && configDockerfilePath) {
-            context.dockerfilePath = path.join(rootPath, configDockerfilePath);
-        }
+        this.configPath = nonNullValueAndProp(context.deploymentConfigurationSettings, this.key as keyof typeof context.deploymentConfigurationSettings)
 
-        const configEnvPath = context.deploymentConfigurationSettings?.envPath;
-        if (!context.envPath && configEnvPath) {
-            context.envPath = path.join(rootPath, configEnvPath);
-        }
-
-        const configSrcPath = context.deploymentConfigurationSettings?.srcPath;
-        if (!context.srcPath && configSrcPath) {
-            context.srcPath = path.join(rootPath, configSrcPath);
+        if (!context[this.key] && this.configPath) {
+            const fullPath = path.join(rootPath, this.configPath);
+            if (await this.verifyFilePath(fullPath)) {
+                context[this.key] = fullPath;
+            }
         }
     }
 
-    public shouldExecute(context: WorkspaceDeploymentConfigurationContext): boolean {
-        return !!context.deploymentConfigurationSettings;
+    public async verifyFilePath(path: string): Promise<boolean> {
+        if (await AzExtFsExtra.pathExists(path)) {
+            return true;
+        } else {
+            throw new Error(localize('fileNotFound', 'File not found: {0}', this.configPath));
+        }
     }
 
     protected createSuccessOutput(_: WorkspaceDeploymentConfigurationContext): ExecuteActivityOutput {
-        // Todo: Finish this, if necessary
-        return {};
+        if (!this.configPath || this.configPath === '') {
+            return {};
+        }
+
+        return {
+            item: new GenericTreeItem(undefined, {
+                contextValue: createActivityChildContext(['filePathVerifyStepSuccessItem', activitySuccessContext]),
+                label: localize('verifyPath', 'Verify {0} path', this.fileType),
```

Comment:
Would love to also see the file path here as well

Example:
`Verify src path "src"`
`Verify dockerfile path "src/Dockerfile"`

Reply:
I thought about that but then went against it since the path name could be long. It's unlikely it would be but might look a little ugly if it is.

Reply:
Yeah that's fair.  I'll leave this one up to you to make the final call then 👍

Reply:
Yeah you're right, it could look busy if the user has long path names.  Let's keep it as is then :)